tools/crm_resource.c

```diff
@@ -1841,7 +1832,7 @@ main(int argc, char **argv)
             if (options.cmdline_config) {
                 exit_code = cli_resource_execute_from_params(out, NULL,
                     options.v_class, options.v_provider, options.v_agent,
-                    "validate-all", options.cmdline_params,
+                    options.operation, options.cmdline_params,
```

Comment:
Aha, good catch. The --class/--provider/--agent/--option options were added in 2.0.2 and limited to --validate (crm_resource would error if they were used without --validate). As of 2.0.5, as a side effect of changing the option processing, we no longer error if those are given with a --force-* option instead of --validate, but since validate-all is hardcoded, it wouldn't have worked.

I believe this will work to allow --force-* to use the options, which seems like a good idea. We should also update the error message in validate_cmdline_config().